### DISCUSSION TITLE
Payment issues.

### DISCUSSION THREAD

**Discussion by 17621859515** [MystenLabs/sui#7881] [Q&A]
The following code on the devnet network sends sui, and the error in the picture will be reported.
How should it be solved?
`
import { Ed25519Keypair, JsonRpcProvider, RawSigner } from '@mysten/sui.js';
// Generate a new Ed25519 Keypair
const keypair = new Ed25519Keypair();
const provider = new JsonRpcProvider();
const signer = new RawSigner(keypair, provider);
const transferTxn = await signer.transferObject({
  objectId: '0x5015b016ab570df14c87649eda918e09e5cc61e0',
  gasBudget: 1000,
  recipient: '0xd84058cb73bdeabe123b56632713dcd65e1a6c92',
});
console.log('transferTxn', transferTxn);
`

![image](https://user-images.githubusercontent.com/69946784/215437007-0c71d649-6ce9-4057-868d-decab2a535ee.png)

**Comment by babs00-hub** [comment]
Àwa
